mpa/modules/models/classifiers/sam_classifier.py

```diff
@@ -78,7 +78,8 @@ def forward_train(self, img, gt_label, **kwargs):
         x = self.extract_feat(img)
 
         losses = dict()
-        if kwargs.get('img_metas', False) and len(kwargs['img_metas'][-1]['ignored_labels']) > 0:
+
+        if self.multilabel:
             loss = self.head.forward_train(x, gt_label, **kwargs)
         else:
             loss = self.head.forward_train(x, gt_label)
```

Comment:
Why do we need this condition by the way? The prototype allows extra params anyway.
![image](https://user-images.githubusercontent.com/4629606/176823917-09571c59-7edc-4b92-8657-542cec93e270.png)

Reply:
In mpa, multi-class heads don't allow extra params.
https://github.com/open-mmlab/mmclassification/blob/v0.9.0/mmcls/models/heads/linear_head.py#L53
https://github.com/openvinotoolkit/model_preparation_algorithm/blob/develop/mpa/modules/models/heads/non_linear_cls_head.py#L85

But above 0.18.0 version for mmcls, linear_head also allows extra params.
https://github.com/open-mmlab/mmclassification/blob/v0.18.0/mmcls/models/heads/linear_head.py#L49

I think this condition can be removed if we upgrade mmcls version.